[OSDOCS#16946]: Make changes as per the errors and warnings for CQA-2 (#114626)
openshift/openshift-docs: modules/storage-persistent-storage-nfs-volume-security.adoc

[id="nfs-volume-security_{context}"]
= NFS volume security

[role="_abstract"]

> This section covers NFS volume security, including matching permissions and SELinux considerations. The user is expected to understand the basics of POSIX permissions, process UIDs, supplemental groups, and SELinux.

Reviewer: ```suggestion
[role="_abstract"]
To understand NFS volume security, including matching permissions and SELinux considerations, you should understand the basics of POSIX permissions, process UIDs, supplemental groups, and SELinux.
```
